code/modules/mob/living/basic/basic.dm

```diff
@@ -260,3 +260,9 @@
 	else if(on_fire && !isnull(last_icon_state))
 		return last_icon_state
 	return null
+
+/mob/living/basic/get_body_temp_heat_damage_limit()
+	return maximum_survivable_temperature
+
+/mob/living/basic/get_body_temp_cold_damage_limit()
+	return minimum_survivable_temperature
```

Comment:
not a fan, they are meant to be template vars for easy applying.

Reply:
I don't know how else I would do this without calling `GetComponent` and reading the value from the component, which cannot be changed from this value at runtime anyway except through VV.

Reply:
we figured out a compromise on discord